pwndbg/commands/telescope.py

```diff
@@ -167,22 +168,22 @@ def telescope(
         stop = address - ptrsize
         step = -1 * ptrsize
 
-    # Find all registers which show up in the trace, map address to regs
-    regs: dict[int, str] = {}
+    # Find all names which show up in the trace
+    names = {}
     for i in range(start, stop, step):
-        values = list(reg_values[i])
+        values = list(name_values[i])
```

Comment:
Isn't this a list already?

Reply:
@disconnect3d i think its name_values = collections.defaultdict(lambda: [])

what do you think?